README.md

```diff
@@ -105,8 +105,9 @@ Criteria:
 
   * Specific tasks that must have run in each build (if they are part of that build).
 
-  If not criteria are specified, a success threshold of 0.95 will be used.
-
+  If criteria are not specified, a success threshold of 0.95 will be used and
+  all other 3 options will have a default value of none.
```

Comment:
[nit] 
`If no criteria are specified`
`and the other 3 above options will be null.`